Minio/MinioClient.cs

```diff
@@ -339,6 +339,17 @@ public MinioClient WithProxy(IWebProxy proxy)
             return this;
         }
 
+        /// <summary>
+        /// Uses the set timeout for all requests if this method is invoked on client object
+        /// </summary>
+        /// <param name="timeout">Timeout in miliseconds.</param>
```

Comment:
```suggestion
        /// <param name="timeout">Timeout in milliseconds.</param>
```

Reply:
Thank you! I applied your suggested change.